src/pages/product/product.css

```diff
@@ -0,0 +1,217 @@
+:root {
+    /* COLORS */
+    --clr-neutral-000: #fff;
+    --clr-neutral-100: rgb(236, 236, 236);
+    --clr-neutral-200: #ddd;
+    --clr-neutral-600: #666;
+    --clr-neutral-900: #000;
+  
+    --clr-primary-500: #007bff;
+  
+    /* FONT FAMILIES */
+    --ff-sans: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+      Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue",
+      sans-serif;
+  
+    /* FONT SIZES */
+    --fs-normal: 1rem;
+    --fs-small: 0.9rem;
+    --fs-xs: 0.8rem;
+  }
+  
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+  
+  body {
+    font-family: var(--ff-sans);
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    background-color: var(--clr-neutral-000);
+  }
+  
+  .product-container {
+    display: flex;
+    background-color: var(--clr-neutral-000);
+    gap: 30px;
+    margin: auto;
+    padding: 5rem 0;
+  }
+  
+  .product-image {
```

Comment:
just to be safe so that no styles get overwritten (in case some of these classnames have been used in other css files) you can nest all of the following lines (46:217) into the .product-container like this example: 
.product-container {
    .product-image {
        ...
    }
}